docs/docs/self-host/docker/run-locally.md

```diff
@@ -4,7 +4,7 @@ title: Self-host OpenSign™ with Docker on Linux/MacOS/Windows
 import ReactPlayer from 'react-player';
 
 ## Steps to run OpenSign™ with docker on Linux/MacOS/Windows:
-  1. Firstly, install [Docker](https://www.docker.com/products/docker-desktop/) and ensure it is running before proceeding to the next steps. Make sure that you have [NodeJS](https://nodejs.org/en) & [GIT](https://git-scm.com/downloads) installed on your machine.
```

Comment:
why do you want to remove that as NPM is mandatory ?

Reply:
As far as I can tell you don't need to install NodeJS or Git if you're doing the Docker install as everything it contained within the container?